docs/options.md

```diff
@@ -7,24 +7,24 @@
 For the available parsers, take a look at: [parsers](parser). For instance:
 
 ```console
-$ conftest test -p examples/hcl2/policy examples/hcl2/terraform.tf -i hcl2
-FAIL - examples/hcl2/terraform.tf - ALB `my-alb-listener` is using HTTP rather than HTTPS
-FAIL - examples/hcl2/terraform.tf - ASG `my-rule` defines a fully open ingress
-FAIL - examples/hcl2/terraform.tf - Azure disk `source` is not encrypte
+$ conftest test -p examples/hcl/policy examples/hcl/terraform.tf -i hcl
+FAIL - examples/hcl/terraform.tf - ALB `my-alb-listener` is using HTTP rather than HTTPS
+FAIL - examples/hcl/terraform.tf - ASG `my-rule` defines a fully open ingress
+FAIL - examples/hcl/terraform.tf - Azure disk `source` is not encrypted
 ```
 
 The `--input` flag can also be a good way to see how different input types would be parsed:
 
 ```console
-conftest parse examples/hcl2/terraform.tf -i hcl2
+conftest parse examples/hcl/terraform.tf -i hcl
```

Comment:
I think it would better showcase the input flag if the input in the example wasn't the default behavior. If `conftest` sees a `.hcl` or `.tf` file, `-i hcl` is to be assumed (which is now the hcl2 parser).

I would actually love to see us renaming the `terraform` parser to `hcl`, because thats what its really doing.

Then in the `examples`, we could have `hcl` and/or `hcl2` and/or `terraform-hcl1` and/or `terraform-hcl2`. Not advocating for every single combination, but thats the idea anyway. HCL and HCL2 being the things we parse, and Terraform being an example, but Terraform can be written using HCL1 or HCL2.

Reply:
agreed on general, but better to rename `terraform` to `hcl1` instead of `hcl`.  Because the `hcl` input references to the `hcl2`. Using hcl here could be confusing